apps/src/templates/census2017/YourSchool.jsx

```diff
@@ -107,9 +107,10 @@ class YourSchool extends Component {
               Does your school teach Computer Science?
             </h1>
             <h3 style={styles.description}>
-              Find your school on the map to see if computer science is already
-              being offered. Can't find your school on the map?{' '}
-              <a href="#form">Fill out the survey below</a>.
+              Find your school on the map to see if computer science was offered
+              during the 2021-2022 school year. Then{' '}
```

Comment:
With the year references are their existing constants we could control these by? How do we manage other year references on the page?

Reply:
Good point, thank you! I'll match it to how it's obtained for the rest of the page: the current census year is passed in and it'll have like {year - 1} or {year + 1} to adjust for writing the school year range like "2022-2023."